src/components/accessibility/screen_reader.tsx

```diff
@@ -3,12 +3,19 @@ import classNames from 'classnames';
 
 export interface EuiScreenReaderOnlyProps {
   children: ReactElement<any>;
+
+  /**
+   * For keyboard navigaiton, force content to display visually upon focus.
+   */
+  showOnFocus?: boolean;
 }
 
 export const EuiScreenReaderOnly: FunctionComponent<
   EuiScreenReaderOnlyProps
-> = ({ children }) => {
-  const classes = classNames('euiScreenReaderOnly', children.props.className);
+> = ({ children, showOnFocus }) => {
+  const classes = classNames('euiScreenReaderOnly', children.props.className, {
+    'euiScreenReaderOnly--showOnFocus': showOnFocus,
+  });
```

Comment:
Then this logic would be more like:

```suggestion
  const classes = classNames({
    'euiScreenReaderOnly': !showOnFocus,
    'euiScreenReaderOnly--showOnFocus': showOnFocus,
  }, children.props.className);
```